## 防火墙
| 日期 | 状态 | 症状 |
| ------ | ------ | ------ |
| 2020-10-19 | 失效 |请求目标服务器时被截断链接 |
| x | x |x |

- 2020-10-19
```
广州电信

(.py3) pro:shadowsocks-over-websocket play$ node local.js -s  xxxx.herokuapp.com -l 1080 -m mmmmm -k password -p 80
[2020-11-20 09:21:26.705] [INFO] local - local is listening on 127.0.0.1:1080

这时在另外一个terminal测试
curl --socks5 localhost:1080 http://httpbin.org/ip
返回
curl: (52) Empty reply from server


[2020-11-20 09:21:30.923] [INFO] local - [1]: accept connection from client
[2020-11-20 09:21:30.929] [INFO] local - [1]: connecting to 34.198.212.59:80
[2020-11-20 09:21:32.489] [ERROR] local - [1]: an error of server connection occured { inspect: [Function: inspect] }
[2020-11-20 09:21:32.493] [INFO] local - [1]: close event[code = 'reserved for extensions'] of server connection has been triggered
[2020-11-20 09:21:32.505] [INFO] local - [1]: end event of client connection has been triggered
[2020-11-20 09:21:32.506] [INFO] local - [1]: close event[had error = false] of client connection has been triggered

但，使用手机网络来测试，一切正常
```
